packages/core/src/common/classes.ts

```diff
@@ -205,7 +205,6 @@ export const TAB_PANEL = `${TAB}-panel`;
 export const TABS = `${TAB}s`;
 
 export const TAG = `${NS}-tag`;
-export const TAG_REMOVABLE = `${TAG}-removable`;
```

Comment:
another breaking change to make note of in release notes

Reply:
def, tho it only affects CSS consumers. #2587